### DISCUSSION TITLE
How to select lines from the current point to a few lines before the end of the file?

### DISCUSSION THREAD

**Discussion by brabalan** [meow-edit/meow#572] [Q&A] [answered]
I just had a selection puzzle that I could not solve, so I guess I’m missing some tricks… I’m in the middle of a huge file, and I need to select all the lines between the current one till a few lines before the end. Using vi, I would switch on visual line, go to the end of the file, and go up a few lines. How can I do this with meow?

**Comment by DogLooksGood** [comment, marked as answer]
I think you want to start the selection with `meow-right-expand`, then go to the end of buffer with `M->`, then move cursor up a few lines.

**Reply by brabalan** [reply]
Ah, thank you. I don’t think it’s in the tutorial. I’ll propose a PR when I find the time.
